data/plugins/linewrapping.lua

```diff
@@ -64,7 +66,14 @@ config.plugins.linewrapping = common.merge({
       path = "require_tokenization",
       type = "toggle",
       default = false
-    }
+    },
+    {
+    label = "Files",
+    description = "List of Lua patterns matching files extensions to linewrap.",
```

Comment:
```suggestion
    description = "List of Lua patterns matching file names to linewrap.",
```

I'd also mention this config in the description of `enable_by_default`.